src/SablierV2LockupTranched.sol

```diff
@@ -0,0 +1,303 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity >=0.8.22;
+
+import { IERC20 } from "@openzeppelin/contracts/token/ERC20/IERC20.sol";
+import { SafeERC20 } from "@openzeppelin/contracts/token/ERC20/utils/SafeERC20.sol";
+import { ERC721 } from "@openzeppelin/contracts/token/ERC721/ERC721.sol";
+import { UD60x18 } from "@prb/math/src/UD60x18.sol";
+
+import { SablierV2Lockup } from "./abstracts/SablierV2Lockup.sol";
+import { ISablierV2Comptroller } from "./interfaces/ISablierV2Comptroller.sol";
+import { ISablierV2LockupTranched } from "./interfaces/ISablierV2LockupTranched.sol";
+import { ISablierV2NFTDescriptor } from "./interfaces/ISablierV2NFTDescriptor.sol";
+import { Helpers } from "./libraries/Helpers.sol";
+import { Lockup, LockupTranched } from "./types/DataTypes.sol";
+
+/*
+
+███████╗ █████╗ ██████╗ ██╗     ██╗███████╗██████╗     ██╗   ██╗██████╗     
+██╔════╝██╔══██╗██╔══██╗██║     ██║██╔════╝██╔══██╗    ██║   ██║╚════██╗    
+███████╗███████║██████╔╝██║     ██║█████╗  ██████╔╝    ██║   ██║ █████╔╝    
+╚════██║██╔══██║██╔══██╗██║     ██║██╔══╝  ██╔══██╗    ╚██╗ ██╔╝██╔═══╝     
+███████║██║  ██║██████╔╝███████╗██║███████╗██║  ██║     ╚████╔╝ ███████╗    
+╚══════╝╚═╝  ╚═╝╚═════╝ ╚══════╝╚═╝╚══════╝╚═╝  ╚═╝      ╚═══╝  ╚══════╝    
+
+██╗      ██████╗  ██████╗██╗  ██╗██╗   ██╗██████╗    ████████╗██████╗  █████╗ ███╗   ██╗ ██████╗██╗  ██╗███████╗██████╗ 
+██║     ██╔═══██╗██╔════╝██║ ██╔╝██║   ██║██╔══██╗   ╚══██╔══╝██╔══██╗██╔══██╗████╗  ██║██╔════╝██║  ██║██╔════╝██╔══██╗
+██║     ██║   ██║██║     █████╔╝ ██║   ██║██████╔╝      ██║   ██████╔╝███████║██╔██╗ ██║██║     ███████║█████╗  ██║  ██║
+██║     ██║   ██║██║     ██╔═██╗ ██║   ██║██╔═══╝       ██║   ██╔══██╗██╔══██║██║╚██╗██║██║     ██╔══██║██╔══╝  ██║  ██║
+███████╗╚██████╔╝╚██████╗██║  ██╗╚██████╔╝██║           ██║   ██║  ██║██║  ██║██║ ╚████║╚██████╗██║  ██║███████╗██████╔╝
+╚══════╝ ╚═════╝  ╚═════╝╚═╝  ╚═╝ ╚═════╝ ╚═╝           ╚═╝   ╚═╝  ╚═╝╚═╝  ╚═╝╚═╝  ╚═══╝ ╚═════╝╚═╝  ╚═╝╚══════╝╚═════╝ 
+
+*/
+
+/// @title SablierV2LockupTranched
+/// @notice See the documentation in {ISablierV2LockupTranched}.
+contract SablierV2LockupTranched is
+    ISablierV2LockupTranched, // 1 inherited component
```

Comment:
This should be "6 inherited components".

+ ISablierV2LockupTranched
 -> ISablierV2Lockup
, [IERC721Metadata, IERC721, IERC165, ISablierV2Base, IAdminable]